examples/VerticalWithinHorizontalScrolling/Sample/GradientTableNode.mm

```diff
@@ -58,7 +58,8 @@ - (ASCellNode *)tableNode:(ASTableNode *)tableNode nodeForRowAtIndexPath:(NSInde
 {
```

Comment:
We should migrate to the `-nodeBlock` data source method as well. Not a blocker for this PR though.

Reply:
I find there're a few other example projects which also need to migrate to the `-nodeBlock` data source method, will create a new PR for fixing this issue altogether :)

Reply:
hi @nguyenhuy , the PR is ready at #1596